src/pages/references/authoring/editor/classes/LineNode.md

```diff
@@ -22,19 +22,19 @@ A LineNode represents a simple line object in the scenegraph – a single straig
 
 ### Accessors
 
-- [absoluteRotation](LineNode.md#absoluteRotation)
-- [absoluteTransform](LineNode.md#absoluteTransform)
-- [allChildren](LineNode.md#allChildren)
-- [blendMode](LineNode.md#blendMode)
-- [endArrowHeadType](LineNode.md#endArrowHeadType)
+- [absoluteRotation](LineNode.md#absoluterotation)
+- [absoluteTransform](LineNode.md#absolutetransform)
+- [allChildren](LineNode.md#allchildren)
+- [blendMode](LineNode.md#blendmode)
+- [endArrowHeadType](LineNode.md#endarrowheadtype)
 - [endX](LineNode.md#endX)
 - [endY](LineNode.md#endY)
```

Comment:
Are these camelCase anchor tag names a problem for the docs site? There's also "startX" and "startY" below... and all nodes have "translateX" and "translateY" too... I think those will all be broken links.